server/src/lib/files.ts

```diff
@@ -15,9 +14,19 @@ const CONFIG_PATH = path.resolve(__dirname, '../../..', 'config.json');
 const config = require(CONFIG_PATH);
 const BUCKET_NAME = config.BUCKET_NAME || 'common-voice-corpus';
 
+var AWS = require('aws-sdk');
+
+if(process.env.HTTP_PROXY) {
+  var proxy = require('proxy-agent');
+
+  AWS.config.update({
+    httpOptions: { agent: proxy(process.env.HTTP_PROXY) }
+  });
+}
+
```

Comment:
Since we do essentially the same thing twice here, any chance, I could get you to create a new module. Maybe something like:
`server/src/lib/aws.ts`
```
var AWS = require('aws-sdk');

if(process.env.HTTP_PROXY) {
  var proxy = require('proxy-agent');

  AWS.config.update({
    httpOptions: { agent: proxy(process.env.HTTP_PROXY) }
  });
}

module.exports = AWS
```
Then, both here and in clip.ts, you could just:
`const AWS = require('./aws');`